Tasks/SqlServerDacpacDeployment/DeployToSqlServer.ps1

```diff
@@ -50,6 +50,11 @@ if( $publishProfile -eq $env:SYSTEM_DEFAULTWORKINGDIRECTORY -or $publishProfile
 
 $sqlDeploymentScriptPath = Join-Path "$env:AGENT_HOMEDIRECTORY" "Agent\Worker\Modules\Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs\Scripts\Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Sql.ps1"
 
+if( -Not (Test-Path -Path $sqlDeploymentScriptPath))
+{
+    $sqlDeploymentScriptPath = Join-Path "$env:AGENT_HOMEDIRECTORY" "externals\vstshost\Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs\Scripts\Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Sql.ps1"
```

Comment:
This fix assumes certain path requirements in Agent. Is this safe and right direction for us to move?